### ISSUE TITLE
Export all data from SQL to CSV.

### ISSUE THREAD

**Issue by bepitulaz** [Tanibox/tania-core#89]
Create a new menu to export all data from the SQL database to CSV, so that users can import that data to other software easily.

**Comment by drwm-base** [comment]
Can I work on this issue?

**Comment by bepitulaz** [comment]
@drwm-base Yes, you can. Please, move the card to in progress and assign yourself. Thank you.

**Comment by drwm-base** [comment]
Hey @bepitulaz, I can't move the card to in progress or assign myself. Do you need to add me to the community or am I missing something?

**Comment by bepitulaz** [comment]
@drwm-base ok, I've assigned it to you.

**Comment by mrifqip29** [comment]
hello, @bepitulaz is this issue already solved?

**Comment by pablojimpas** [comment]
Hi @mrifqip29, this feature request hasn't been implemented yet. Feel free to make this contribution yourself and I will be happy to review it. If you can't do it yourself and I see more interest in this issue, I may work on it myself.

**Comment by mrifqip29** [comment]
HI @pablojimpas, sure I will try to work on it. Thank you

**Comment by pablojimpas** [comment]
> HI @pablojimpas, sure I will try to work on it. Thank you

I've assigned this issue to you. Thanks for your interest, please ask for help if you need it.
